tcms/core/models/base.py

```diff
@@ -11,4 +11,4 @@ class UrlMixin:  # pylint: disable=too-few-public-methods
     def get_full_url(self):
         site = Site.objects.get(pk=settings.SITE_ID)
         host_link = request_host_link(None, site.domain)
-        return '{}/{}/'.format(host_link, self._get_absolute_url().strip('/'))
+        return '{}/{}'.format(host_link, self._get_absolute_url().strip('/'))
```

Comment:
This method is used for multiple classes and all of them require the trailing slash. TestPlan is different that it supports 2 URLs for the same object and that doesn't quite fit here.